connectors/src/connectors/github/temporal/client.ts

```diff
@@ -68,7 +68,7 @@ export async function launchGithubFullSyncWorkflow({
     return;
   }
 
-  await client.workflow.start(githubFullSyncWorkflow, {
+  await client.workflow.start(githubSyncAllReposWorkflow, {
```

Comment:
IMO we should keep the name "full sync" that's commonly used across the eng team to refer to... full syncs

we could go `githubFullSync`, ghFullSyncWorkflow, fullSyncWorkflow which are used elsewhere

Reply:
in this file since we have some other workflows untouched that start with `github` and end with `workflow` I want to keep that

Reply:
let's do v2 👍